res/values-pt-rBR/strings.xml

```diff
@@ -1018,8 +1018,8 @@ Se você notar um jogador quebrando as regras do jogo, reporte á um Moderador.
     <string name="Save_the_profile_first">Salve o perfil primeiro</string>
     <string name="Birthday">Aniversário</string>
     <string name="Mass_Boost">Impulso de Massa</string>
-    <string name="Mass_Boost_does_not_work_in_arenas_and_tournaments_">Impulso de Massa não funciona em arenas, torneios, e Guerras de Clãs!</string>
-    <string name="ArenaDescriptionPure">SEM autoclick ou ultraclick. Você terá apenas uma vida. O vencedor receberá %s plasma. Negociar wins ou "free wins" resultará em um banimento.</string>
+    <string name="Mass_Boost_does_not_work_in_arenas_and_tournaments_">O Impulso de Massa não funciona em arenas, torneios, e Guerras de Clãs!</string>
+    <string name="ArenaDescriptionPure">SEM autoclick ou ultraclick. Você terá apenas uma vida. O vencedor receberá %s plasma. Negociar vitórias ou "free wins" resultará em um banimento.</string>
```

Comment:
same here, I would find a translation for "free wins" (pontos grátis, or however you find better) and keep it translated throughout the text